app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/LoginResponse.kt

```diff
@@ -10,22 +10,17 @@ import com.google.gson.annotations.SerializedName
  */
 
 class LoginResponse {
```

Comment:
Make this data class

Reply:
@iamareebjamal  Considering the context of this PR, should I make this change in this PR itself or should I send another PR for this?

Reply:
OK